requirements.txt

```diff
@@ -1,2 +1,4 @@
 Flask
-waitress
\ No newline at end of file
+waitress
+thefuzz
+os
```

Comment:
The `os` import that you used in your code is not an external dependency, but rather, it is built in to Python. Because of this, an entry for it is not needed in `requirements.txt`.